R/plot_xctr_company_level.R

```diff
@@ -0,0 +1,48 @@
+#' Plot xctr data on a company level
+#'
+#' @param data A data frame like [xctr_toy_data].
+#' @param company_name A string. Name of one company in the data set.
+#'
+#' @return A [ggplot] object.
+#' @export
+#'
+#' @examples
+#' plot_xctr_company_level(xctr_toy_data, "company_a")
+plot_xctr_company_level <- function(data, company_name) {
+
+  data <- data |>
+    na.omit() |>
+    filter(company_name == {{ company_name }})
```

Comment:
If `compamy_name` is a string, then you may not need the weird `{{ }}`? But maybe that's the new recommended way to do it? I think here you want to differenciate from `.data$company_name`, in which case you might use `.env$company_name` (if that's still the best way) or instead use `.company_name <- company_name` then use `filter(company_name == .company_name)`.